build.gradle

```diff
@@ -24,17 +24,17 @@ ext {
 
     // Dependencies
     reactor_bom_version = '2020.0.33'
-    jackson_databind_version = '2.12.7.1'
-    jackson_datatype_jsr310_version = '2.14.2'
-    jackson_datatype_jdk8_version = '2.12.7'
-    caffeine_version = '2.8.8'
+    jackson_databind_version = '2.17.0'
+    jackson_datatype_jsr310_version = '2.17.0'
+    jackson_datatype_jdk8_version = '2.17.0'
+    caffeine_version = '2.9.3'
     immutables_group = 'org.immutables'
-    immutables_version = '2.9.0'
+    immutables_version = '2.10.1'
 
     // Test dependencies
     junit_version = '5.7.1'
     mockito_version = '3.11.2'
-    logback_version = '1.2.3'
+    logback_version = '1.5.6'
```

Comment:
Only logback versions < 1.4.X are compatible with java 8 (https://logback.qos.ch/dependencies.html)

Reply:
/cc @Doc94

Reply:
I have a fix up. I found it in their Github [README.md](https://github.com/qos-ch/logback/blob/master/README.md)

> The 1.3.x series requires Java 8 at runtime. If you wish to build logback from source, you will need Java 9.